homeassistant/components/light/hue.py

```diff
@@ -384,7 +384,7 @@ def turn_on(self, **kwargs):
                 hue, sat = color_util.color_xy_to_hs(*kwargs[ATTR_XY_COLOR])
                 command['hue'] = hue
                 command['sat'] = sat
-                command['bri'] = self.info['bri']
+                command['bri'] = self.info.get('bri')
```

Comment:
I think this is setting the brightness to the value that HA has cached, right? Can we simply delete the line and the bulb will maintain the current brightness (but change its hue and saturation)?